third_party/move/move-ir-compiler/move-ir-to-bytecode/syntax/src/syntax.rs

```diff
@@ -1483,6 +1483,7 @@ fn parse_acquire_list(
     Ok(al)
 }
 
+#[allow(clippy::four_forward_slashes)]
 //// Spec language parsing ////
```

Comment:
Why not use 2?  A few of the other ones your converted from 4 to 3 might have been converted to unexpected doc comments

Reply:
SGTM. 😄 I'll convert them to 2 -- kinda figured folks wanted docs comments (except in this particular case), but I'll use 2 and let them update.